pkg/ccl/streamingccl/streamingest/stream_ingestion_job.go

```diff
@@ -536,8 +536,16 @@ func (s *streamIngestionResumer) CollectProfile(ctx context.Context, execCtx int
 		defer s.mu.Unlock()
 		aggStatsCopy = s.mu.perNodeAggregatorStats.DeepCopy()
 	}()
-	return bulkutil.FlushTracingAggregatorStats(ctx, s.job.ID(),
-		p.ExecCfg().InternalDB, aggStatsCopy)
+	if err := bulkutil.FlushTracingAggregatorStats(ctx, s.job.ID(),
+		p.ExecCfg().InternalDB, aggStatsCopy); err != nil {
```

Comment:
should we instead log errors if any of these independent profiling tasks fail? I.e. if the TracingAggregatorStats call fails, it would still be nice to try get the FrontierExectionDetailFile

Reply:
good point, I still think I want the final error to bubble up to the client so that we know something is wrong but it makes sense to do everything we can before that. How do you like this?